german/Sonnenbrand/vor-sonne-schutzen.md

```diff
@@ -0,0 +1,4 @@
+Sie sind aus dem Schlaf erwacht und merken das Brennen der Haut in ihrem Gesicht. Es ist Ihnen nicht möglich die Gesichtszüge zu verändern ohne höllichen Schmerz wahrzunehmen. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.